riscv-sbi.adoc

```diff
@@ -1674,6 +1675,259 @@ The possible error codes returned in `sbiret.error` are shown in the
 | sbi_pmu_counter_fw_read         | 0.3         | 5   | 0x504D55
 |===
 
+== Attestation and Measurement Extension (EID #0x41545354 "ATST")
+
+The Attestation and Measurement (ATST) Extension provides a set of functions for
+implementing TCB attestation flows and for extending TCB measurements. The SBI
+implementation composes those measurements into an attestation evidence that can
+be used by attestation verifiers, either locally or remotely.
+
```

Comment:
```suggestion
                                  Layer n+1
                                 +--------------------------------------------+
                                 | Extend Measurements for layer n+2          |
                                 | Generate evidence cert (CDI) for layer n+2 |
                                 | Access-control current layer keys          |
                                 | Transfer control to layer n+2              |
                                 +-------^------------------------------------+
                                         |
  Layer n                                |
+--------------------------------------------+
| Extend Measurements for layer n+1          |
| Generate evidence cert (CDI) for layer n+1 |
| Access-control current layer keys          |
| Transfer control to layer n+1              |
+--------------------------------------------+
```

Reply:
Added a couple of diagrams in the intro section. Please let me know how it looks.